custom_components/powercalc/data/ikea/LED1949C5/model.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
color_temp.csv also has some 0.0 values, but only a few. You may manually edit these to 0.3 watt. No need to redo this measurements with multiple bulbs.

Reply:
OK. Then I just redo the measurement for the colored ones, the hs mode.

Reply:
Yes that's fine.